pyfms/utils/ctypes.py

```diff
@@ -0,0 +1,155 @@
+from ctypes import CDLL, POINTER, c_bool, c_char_p, c_double, c_float, c_int
```

Comment:
Is this the system ctypes or the pyfms package ctypes?

Reply:
system